packages/e2e/e2e-base/playwright.config.ts

```diff
@@ -18,9 +18,7 @@ export const baseConfig: PlaywrightTestConfig = {
     trace: 'retain-on-failure',
     viewport: { width: 1920, height: 1080 },
   },
-  timeout: 100000,
-
   expect: {
-    timeout: 2 * 30000,
+    timeout: 15000,
```

Comment:
🔥

Reply:
Now that we pre-compile apps, this is finally possible. This is particularly useful when a test fails, we don't have to wait for this long for feedback.